test/TripRepo-test.js

```diff
@@ -42,10 +42,52 @@ describe('TripRepo', () => {
 
   describe('getFolioByYear', () => {
 
-    it('should return a new TripRepo of all the trips that match a given userID', function() {
+    it('should return a new TripRepo of all the trips that match a given userID', () => {
 
       expect(repo.getFolioByYear(2019)).to.deep.equal(new TripRepo(data, []));
       expect(repo.getFolioByYear(0)).to.deep.equal(new TripRepo([], []));
     });
   });
+
+  describe.only('getPastFolio', () => {
+
+    it('should return a new TripRepo of all the trips that have ended already', () => {
+
+      let date = new Date(2019, 8, 25);
+      expect(repo.getPastFolio(date)).to.deep.equal(repo);
+
+      let bad = new Date(2019, 8, 24);
+      expect(repo.getPastFolio(bad)).to.deep.equal(new TripRepo([], []));
+    });
+  });
+
+  describe.only('getCurrentFolio', () => {
+
+    it('should return a new TripRepo of all the trips that have ended already', () => {
+
+      let beg = new Date(2019, 8, 16);
+      let middle = new Date(2019, 8, 20);
+      let end = new Date(2019, 8, 24);
+      expect(repo.getCurrentFolio(beg)).to.deep.equal(repo);
```

Comment:
I feel like you should test for the object itself just to make sure if is giving you the right values, maybe assign the repo a property or something just to be super sure you're getting the right thing. I would even make your test data a little bigger so you can be sure it is doing the filtering correctly.